### PR TITLE
DOCS-1680 Synthetics Multistep API Tests Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds a reference anchor link to the Use variables section, replaces a screenshot of the Create Your First Step workflow, and rewords two headers to be consistent with the UI. 

### Motivation

DOCS-1680

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-multistep-variable-reference-link/synthetics/multistep

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/multistep.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ Multistep API tests can run from Datadog [managed locations][4] and [private loc
...
 Select the **Locations** to run your Multistep API test from: Multistep API tests can run from both [managed][4] and [private locations][5] depending on whether you are willing to run the test from outside or inside your network.
 
-### Define requests
+### Define steps
 
-Hit **Create Your First Request** to start designing your test's requests.
+To create an HTPP request step, click **Create Your First Step**.
 
-{{< img src="synthetics/api_tests/create_request.png" alt="Create your Multistep API test requests"  style="width:90%;" >}}
+{{< img src="synthetics/api_tests/create_request2.png" alt="Create your Multistep API test requests"  style="width:90%;" >}}
```
Comment (ruthnaebeck) [inline content/en/synthetics/multistep.md:51]:
```suggestion
{{< img src="synthetics/api_tests/create_request2.png" alt="Create your Multistep API test requests" style="width:90%;" >}}
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Review by ruthnaebeck — APPROVED** [review]

File: content/en/synthetics/multistep.md [hunk 2]
```diff
@@ -44,11 +44,11 @@ Multistep API tests can run from Datadog [managed locations][4] and [private loc
 
 Select the **Locations** to run your Multistep API test from: Multistep API tests can run from both [managed][4] and [private locations][5] depending on whether you are willing to run the test from outside or inside your network.
 
-### Define requests
+### Define steps
 
-Hit **Create Your First Request** to start designing your test's requests.
+To create an HTPP request step, click **Create Your First Step**.
```
Comment (MargotLepizzera) [inline content/en/synthetics/multistep.md:49]:
```suggestion
To create an HTTP request step, click **Create Your First Step**.
```

File: content/en/synthetics/multistep.md [hunk 3]
```diff
@@ -44,11 +44,11 @@ Multistep API tests can run from Datadog [managed locations][4] and [private loc
...
 
-{{< img src="synthetics/api_tests/create_request.png" alt="Create your Multistep API test requests"  style="width:90%;" >}}
+{{< img src="synthetics/api_tests/create_request2.png" alt="Create your Multistep API test requests" style="width:90%;" >}}
```
Comment (MargotLepizzera) [inline content/en/synthetics/multistep.md:51]: Ok weird feedback but that image seems more blurry than the existing one, any idea why?

File: content/en/synthetics/multistep.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ You can create up to 20 assertions per step by clicking on **New Assertion** or
 
 {{< img src="synthetics/api_tests/assertions.png" alt="Define assertions for your Multistep API test" style="width:90%;" >}}
 
-##### Failure behavior
+#### Add execution parameters
```
Comment (MargotLepizzera) [inline content/en/synthetics/multistep.md:124]: Good point!

File: content/en/synthetics/multistep.md [hunk 5]
```diff
@@ -143,7 +143,7 @@ To parse your variable:
 
 {{< img src="synthetics/api_tests/ms_extract_variable.png" alt="Extract variables from HTTP requests in Multistep API test" style="width:90%;" >}}
 
-Once created this variable can be used in the following steps of your Multistep API test.
+Once created, this variable can be used in the following steps of your Multistep API test. For more information, see [Use variables](#use-variables).
```
Comment (MargotLepizzera) [inline content/en/synthetics/multistep.md:146]: YAS!

**Comment by MargotLepizzera** [comment]
Added a few comments but LGTM overall @alai97! Thanks for the great work 👍

File: content/en/synthetics/multistep.md (hunk 2 above)
Comment (alai97) [reply content/en/synthetics/multistep.md:49]: Great catch!

File: content/en/synthetics/multistep.md (hunk 3 above)
Comment (alai97) [reply content/en/synthetics/multistep.md:51]: This image is definitely sharper than the original, I'm bump up the image width so it appears larger!

Comment (alai97) [reply content/en/synthetics/multistep.md:51]:
```suggestion
{{< img src="synthetics/api_tests/create_request2.png" alt="Create your Multistep API test requests" style="width:100%;" >}}
```

File: content/en/synthetics/settings/_index.md [hunk 6]
```diff
@@ -57,14 +61,18 @@ You can create variables from your existing [HTTP tests][1] by parsing their ass
...
 
-{{< img src="synthetics/settings/variable_fromhttp.png" alt="Variable from http"  style="width:100%;">}}
+Once created, global variables can be used in single and multistep API tests as well as browser tests. For more information, see [Use variables][5].
 
 **Note:** Variable values are updated whenever the test they are extracted from runs.
 
-[1]: /synthetics/api_tests/?tab=httptest
+{{< img src="synthetics/settings/variable_fromhttp.png" alt="Variable from http"  style="width:100%;">}}
+
+
```
Comment (alai97) [inline content/en/synthetics/settings/_index.md:70]:
```suggestion
```

File: content/en/synthetics/settings/_index.md [hunk 7]
```diff
@@ -87,11 +87,13 @@ To generate and use a TOTP in your tests, create a global variable where you ent
 
 {{< /tabs >}}
 
+Once created, global variables can be used in all Synthetic tests by typing `{{` in the field of interest and selecting your global variable. For more information, see individual test documentation pages ([HTTP test][7], [Multistep API test][8], [Browser test configuration][9] and [steps][10]).
```
Comment (alai97) [inline content/en/synthetics/settings/_index.md:90]:
```suggestion
Once created, global variables can be used in all Synthetic tests by typing `{{` in the field of interest and selecting your global variable. For more information, see [HTTP test][7], [Multistep API test][8], [Browser test configuration][9], and [Steps][10].
```

File: content/en/synthetics/settings/_index.md [hunk 8]
```diff
@@ -106,7 +108,7 @@ You can restrict access to a global variable based on the roles in your organiza
 
 ### Default locations
 
-Choose the default locations for your [API test][10], [multistep API test ][11], or [browser test][4] details. 
+Choose the default locations for your [API test][3], [multistep API test ][4], or [browser test][5] details.
```
Comment (alai97) [inline content/en/synthetics/settings/_index.md:111]:
```suggestion
Choose the default locations for your [API test][3], [multistep API test][4], or [browser test][5] details. 
```

**Pull request merged**